develop/connect/use-with-clarity.md

```diff
@@ -246,4 +246,18 @@ const MyComponent = () => {
     <span onClick={onClick}>Call my contract</span>
   );
 };
-```
\ No newline at end of file
+```
+
+## Checking transaction status
+You may want to check the status of a transaction after signing and broadcasting it. For instance, your app might want to enable certain components only _after_ a transaction was successfully processed on the blockchain.
+
+Following the steps above, you'll receive a transaction ID (`txId`) as part of the `finished` callback method once your user signed the transaction. Equipped with the ID, you can use the [Stacks Blockstack API](https://docs.blockstack.org/common/core_ref.html#stacks-blockchain-api) to request the status of the transaction.
+
+```bash
+curl 'https://sidecar.staging.blockstack.xyz/sidecar/v1/tx/<txId>'
+```
```

Comment:
```suggestion
```js
const response = await fetch(`https://sidecar.staging.blockstack.xyz/sidecar/v1/tx/${txId}`)
const txData = await response.json();
console.log(txData.status);
```
```

Reply:
We should use JS instead of curl, because this page is entirely in the context of writing web apps.